py/desispec/workflow/desi_proc_funcs.py

```diff
@@ -354,18 +359,12 @@ def determine_resources(ncameras, jobdesc, queue, nexps=1, forced_runtime=None,
     elif jobdesc == 'NIGHTLYBIAS':
         ncores, runtime = 15, 5
         nodes = 2
-    elif jobdesc == 'TILENIGHT':
-        ncores, nodes = config['cores_per_node'], 1
-        # ~150 frames per node hour plus
-        # 10-minute overhead for a single node
-        # perlmutter-[cpu,gpu] have timefactor=[0.7,0.5]
-        runtime = 18 + int(60. / 140. * ncameras * nexps)
     else:
         msg = 'unknown jobdesc={}'.format(jobdesc)
         log.critical(msg)
         raise ValueError(msg)
 
-    if jobdesc in ('FLAT', 'TESTFLAT', 'SKY',
+    if jobdesc in ('FLAT', 'TESTFLAT', 'SKY', 'TILENIGHT',
         'TWILIGHT', 'SCIENCE','PRESTDSTAR','POSTSTDSTAR'):
         if system_name[0:10] == 'perlmutter':
             ncores, nodes = config['cores_per_node'], 1
```

Comment:
Let's drop this section and encode these choices earlier in the flow so that the `ncores`, `nodes`, and `runtime` are set only once for each jobdesc instead of being set in one place and then possibly reset here.

Reply:
Done. @sbailey please have a look and see if this works.